main.tf

```
@@ -1,13 +1,50 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+locals {
+  enabled = module.this.enabled
+
+  // manage quotas that have a `value` defined
```

The `#` is more appropriate for comments in hcl (according to terraform docs and tflint)

```suggestion
  # manage quotas that have a `value` defined
```